blueprints/periodic-web-poller/src/main.rs

```diff
@@ -1,16 +1,21 @@
 use color_eyre::Result;
 use gadget_sdk::info;
-use gadget_sdk::job_runner::MultiJobRunner;
+use gadget_sdk::runners::tangle::TangleConfig;
+use gadget_sdk::runners::BlueprintRunner;
 use periodic_web_poller_blueprint as blueprint;
 
-#[gadget_sdk::main]
+#[gadget_sdk::main(env)]
 async fn main() {
     let web_poller = blueprint::WebPollerEventHandler {
         client: reqwest::Client::new(),
     };
 
     info!("~~~ Executing the periodic web poller ~~~");
-    MultiJobRunner::new(None).job(web_poller).run().await?;
+    let tangle_config = TangleConfig::default();
+    BlueprintRunner::new(tangle_config, env)
```

Comment:
This is supposed to demonstrate a runner with no env, where the event listener is just a `PeriodicEventListener`.

Reply:
Yes, I implemented this for `()` I forget to fix it here. Thanks for noticing.